sdk/file_sdk/Common/Common.pro

```diff
@@ -4,7 +4,9 @@ TARGET    = modfilecommon
 
 TEMPLATE  = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
+
 INCLUDEPATH += $$REPO_ROOT/sdk/file_sdk
+win32:INCLUDEPATH += $$REPO_ROOT/third_party/include/Libgsf
```

Comment:
>INCLUDEPATH += $$REPO_ROOT/third_par [](start = 6, length = 36)

linux: /usr/include?

Reply:
not needed, these dlls are built in package in linux.

---
In reply to: [139411848](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139411848) [](ancestors = 139411848)